development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+This committee or the TCT_ itself must *explicitly* approve the change.
+In the case of DM Coding Standards, which are change-controlled Confluence pages, the TCT_ has, via `RFC-24 <https://jira.lsstcorp.org/browse/RFC-24>`_, delegated all decision-making to the SA, who must explicitly approve any changes.
+
+Change-controlled documents with other handles, such as LSE- or LPM-, including inter-subsystem interfaces, have project-wide change control processes.
+Please consult the DM PM, SA, or IS for more information.
+
+At least one member of the DM TCT_ will read each RFC to determine if it might affect a change-controlled document.
+
+.. _decision-making-rfc-responsibility:
+
+Responsibility and delegation
+-----------------------------
+
+For project management purposes, RFCs are formally proposals made to the DM PM and PS who by default are responsible for everything in DM (they "own" all problems).
+As owners, they have the final word in accepting or rejecting all proposals.
+Functionally, they delegate that ownership---the right and responsibility to make decisions---to others within the team (e.g. the SA, IS, group leads, etc.) who are expected to delegate it even further.
+Notifying your institutional technical manager about an RFC serves to inform the DM PM.
+
+.. _decision-making-rfd:
+
+Request for Discussion (RFD) Process
+====================================
+
+.. See RFC-53: https://jira.lsstcorp.org/browse/RFC-53
+
+Requests for Discussion (RFD) are intended to facilitate in-depth technical discussions across DM.
+These might be:
+
+- Detailed design discussions for a component of the system or its interfaces.
+- Design reviews for new code or refactorings of old code.
+- Brainstorming methods to solve difficult problems.
+- "Brain dump" explanations of a design to share knowledge across DM.
+
+.. _decision-making-rfd-creating:
+
+Creating an RFD
+---------------
+
+Create an RFD by making a JIRA issue in the `Request For Comments (RFC) <https://jira.lsstcorp.org/projects/RFC>`_ project with a **component of DM** and a **story type of RFD**.
+Use the **Location** field to specify the date and time of the discussion (:ref:`see below for time slot <decision-making-rfd-time>`).
+In the RFD's description:
+
+- Summarize the issue, and indicate a desired outcome from the discussions.
+- Include background material (using JIRA attachments, if necessary).
+- Provide a link to the BlueJeans or Google Hangouts room.
+
+Creating an RFD issue will trigger postings to the `dm-devel mailing list <https://lists.lsst.org/mailman/listinfo/dm-devel>`_ and the `'Bot: RFC' HipChat room <hipchat://hipchat.com/room/1028779>`_.
+
+As the discussion organizer, you are responsible for ensuring that key attendees are available for the time slot.
```

Comment:
Not sure that "key" and "any required" (which should probably be "all required") have the same connotations.